packages/react-ui-core/src/Highlighter/Highlighter.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import { randomId } from '@rentpath/react-ui-utils'
+
+@themed(/^Highlighter/)
+
+export default class Highlighter extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    pattern: PropTypes.string.isRequired,
+    children: PropTypes.string,
+    theme: PropTypes.object,
+    indexHighlighted: PropTypes.number,
+  }
+
+  static defaultProps = {
+    text: '',
+    children: '',
+    theme: {},
+  }
+
+  get indexHighlighted() {
+    return this.props.indexHighlighted + 1
+  }
+
+  renderHighlightedText() {
+    const { pattern, children } = this.props
+
+    if (pattern === '') return children
```

Comment:
I think you just want `if (!pattern) return children` yes?

Reply:
duh. whoops